bundles/org.eclipse.passage.loc.billing.core/src/org/eclipse/passage/loc/billing/internal/core/BillingService.java

```diff
@@ -0,0 +1,14 @@
+package org.eclipse.passage.loc.billing.internal.core;
+
+import java.util.Date;
+import java.util.Set;
+
+import org.eclipse.passage.lic.users.UserLicenseDescriptor;
+
+public interface BillingService {
+
+	Set<String> getUserLicenses(Iterable<? extends UserLicenseDescriptor> licenses);
```

Comment:
the method should return something useful for the further processing, like "how many licenses were issued for the given plan during the given period"  or "how many licenses were issued for the given product during the given period grouped by plans"